documentation/CAMARA-OIDC-profile.md

```diff
@@ -0,0 +1,1022 @@
+
+
+* **nonce**
+
+  String value used to associate a Client session with an ID Token, and to mitigate replay attacks. The value is passed through unmodified from the Authentication Request to the ID Token. If present in the ID Token, Clients MUST verify that the nonce Claim Value is equal to the value of the nonce parameter sent in the Authentication Request. If present in the Authentication Request, Authorization Servers MUST include a nonce Claim in the ID Token with the Claim Value being the nonce value sent in the Authentication Request. Authorization Servers SHOULD perform no other processing on nonce values used. The nonce value is a case-sensitive string.
+
+
+* **acr**
+
+  OPTIONAL. Authentication Context Class Reference. String specifying an Authentication Context Class Reference value that identifies the Authentication Context Class that the authentication performed satisfied.
+
+
+* **amr**
+
+  OPTIONAL. Authentication Methods References. JSON array of strings that are identifiers for authentication methods used in the authentication.
+
+
+* **azp**
+
+  OPTIONAL. Authorized party - the party to which the ID Token was issued. If present, it MUST contain the OAuth 2.0 Client ID of this party. The `azp` value is a case-sensitive string containing a StringOrURI value.
+
+
+* **at_hash**
+
+  OPTIONAL. Access Token hash value. Its value is the base64url encoding of the left-most half of the hash of the octets of the ASCII representation of the access_token value, where the hash algorithm used is the hash algorithm used in the alg Header Parameter of the ID Token's JOSE Header. For instance, if the alg is RS256, hash the access_token value with SHA-256, then take the left-most 128 bits and base64url-encode them. The `at_hash` value is a case-sensitive string.
+
+
+ID Tokens MAY contain other Claims. Any Claims used that are not understood MUST be ignored.
+
+
+## Client Authentication
+
+This section defines a set of Client Authentication methods that are used by Clients to authenticate to the Authorization Server when using the Token or Backchannel Authorization Endpoint.
+
+These Client Authentication methods are:
+
+* **_private_key_jwt_**
+
+  RECOMMENDED. Clients that have registered a public key sign a JWT using that key. The Client authenticates in accordance with JSON Web Token (JWT) Profile for OAuth 2.0 Client Authentication and Authorization Grants [OAuth.JWT] and Assertion Framework for OAuth 2.0 Client Authentication and Authorization Grants [OAuth.Assertions]. At least the Authorization Server MUST support JWS-signed Object with the RS256 algorithm. The JWT MUST contain the following REQUIRED Claim Values and MAY contain the following OPTIONAL Claim Values:
```

Comment:
RFC 7521 should not be referenced here because the relevant sections of that document for client authentication are already referenced by RFC 7523. So RFC 7523 is the only inline reference required.

Reply:
The links have been changed to point to the new RFC 7523 reference.